src/Sarif.Converters/PyLintObjectModel/PyLintLogReader.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.IO;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace Microsoft.CodeAnalysis.Sarif.Converters.PyLintObjectModel
+{
+    public static class PyLintLogReader
+    {
+        public static PyLintLogList ReadLog(string input)
+        {
+            return ReadLog(input, Encoding.UTF8);
+        }
+
+        public static PyLintLogList ReadLog(string input, Encoding encoding)
+        {
+            return ReadLog(new MemoryStream(encoding.GetBytes(input)));
+        }       
+
+        public static PyLintLogList ReadLog(Stream input)
+        {
+            string pyLintText = String.Empty;
```

Comment:
It's not necessary to initialize this. It is unconditionally assigned on Line 26.